### ISSUE TITLE
Interaction with normal window without border

### ISSUE THREAD

**Issue by rekols** [cyberos/cyber-statusbar#2]
I found that some applications do not use system borders. When they are maximized, StatusBar should not display Window Control Buttons.  @omaemae What do you think ?

**Comment by rekols** [comment]
If we want to achieve it, we can judge the "NoBorder" property to determine whether it is a borderless window.

Konsole: 
![Screenshot from 2020-12-18 12-40-25](https://user-images.githubusercontent.com/9636382/102575651-95a74300-412e-11eb-801c-5c49d01dbf82.png)

Chromium:
![Screenshot from 2020-12-18 12-41-10](https://user-images.githubusercontent.com/9636382/102575657-9b048d80-412e-11eb-97ea-0417f8f6611c.png)

**Comment by omaemae** [comment]
It depends, really. Chromium uses a custom border yet it integrates with
Qt/Cyber well.

Applications should implicitly specify that they don't want window buttons,
and the status bar should follow that.

pt., 18 gru 2020, 05:44 użytkownik Reven Martin <notifications@github.com>
napisał:

**Comment by omaemae** [comment]
> Chromium uses a custom border yet it integrates with Qt/Cyber well.

Nevermind, turns out it automatically turned it off when I switched to Cyber.

**Comment by omaemae** [comment]
Yeah, I think we can tell the status bar to not show the window buttons based on lack of border.

**Comment by rekols** [comment]
alright.
